vpdq/cpp/bin/vpdq-hash-video.cpp

```diff
@@ -32,45 +34,39 @@ static void usage(char* argv0, int rc) {
   exit(rc);
 }
 
-// TODO: Move to a shared library
-// Directly copy from TMK
 /**
  *
  * Get the base name with extension of the input path
  *
  * ./dir/sub-dir/sample.txt -> sample.txt
  *
  * @param path Path of the target file
- * @param delimiter The character that marks the beginning or end
  *
  */
-string basename(const string& path, const string& delimiter) {
-  size_t n = path.length();
-  size_t i = path.rfind(delimiter, n);
-  if (i == string::npos) {
+std::string basename(const std::string& path) {
+  size_t i = path.find_last_of("\\/");
```

Comment:
This allows the function to work on Windows and Linux.

"\\/" is used because both backslash and forward slash are special characters in C++, so they need to be escaped with backslashes.